share/instruments/instruments.xml

```diff
@@ -9629,10 +9629,16 @@
                   <shortName>Sh.</shortName>
                   <description>Shamisen</description>
                   <musicXMLid>pluck.shamisen</musicXMLid>
+                  <StringData>
+                        <frets>22</frets>
+                        <string>47</string>
+                        <string>52</string>
+                        <string>59</string>
+                  </StringData>
                   <clef>G8vb</clef>
                   <barlineSpan>1</barlineSpan>
-                  <aPitchRange>50-83</aPitchRange>
-                  <pPitchRange>50-83</pPitchRange>
+                  <aPitchRange>47-83</aPitchRange>
+                  <pPitchRange>47-83</pPitchRange>
```

Comment:
83? How to get there, with highest string being 59 and the number of frets being 22? Makes 81 for me...

Reply:
I'll modify it too
In fact it's because I needed (for test purpose) more glyphs to be displayed, but "in the ideal/final version", the number of fret will be 22 (the limit in my code is 23 to allow the "silence" glyph to be displayed, because I don't found how to "replace" the rest glyph and that in the 3.0, no rest is displayed, even with option enabled)